Gruntfile.js

```diff
@@ -31,7 +31,8 @@ module.exports = function(grunt) {
 
         jshint: {
             options: {
-                'jshintrc': '.jshintrc'
+                'jshintrc': '.jshintrc',
+                'reporterOutput': ''
```

Comment:
@amikolic 
What is function for this "reporterOutput" key?

Reply:
This is added because of error with grunt-contrib-jshint:
`Running "jshint:all" (jshint) task
Warning: Path must be a string. Received null Use --force to continue.`